src/workflows/airqo_etl_utils/satellite_utils.py

```
@@ -1,12 +1,22 @@
+from pathlib import Path
+import zipfile
+import tempfile
 
 import ee
 import numpy as np
 import pandas as pd
+import xarray as xr
 from google.oauth2 import service_account
 
 from airqo_etl_utils.config import configuration
+from airqo_etl_utils.data_sources import DataSourcesApis
```

Ruff (0.11.9)

`airqo_etl_utils.data_sources.DataSourcesApis` imported but unused

Remove unused import: `airqo_etl_utils.data_sources.DataSourcesApis`

(F401)